File: CSHARP/Refactoring/Tests/Dish.cs
namespace RefactoringKungFu
{
    public partial class PastaMakerTests
    {
        public class Dish
        {
            public Dish(SauceType sauce, PastaType pasta)
            {
                this.sauce = sauce;
                this.pasta = pasta;
            }
            public readonly SauceType sauce;
            public readonly PastaType pasta;

            public override string ToString()
            {
                string result = "Sauce: " +
                    sauce.ToString() +
                    " Pasta : " +
                    pasta.ToString();
                return result;

            }
        }

    }
}

File: Java/PastaMaker/src/test/java/pastamaker/Dish.java
package pastamaker;

public class Dish {

	public SauceType sauce;
	public PastaType pasta;

	public Dish(SauceType sauce, PastaType pasta)
    {
        this.sauce = sauce;
        this.pasta = pasta;
    }

    public String toString()
    {
        String result = "Sauce: " +
        				sauce.toString() +
        				" Pasta : " +
        				pasta.toString() + "\n";
        return result;
    }

}
